closure/goog/math/integer.js

```diff
@@ -566,6 +566,60 @@ goog.math.Integer.carry16_ = function(bits, index) {
   }
 };
 
+/**
+ * Returns this Integer divided by the given one.
```

Comment:
This is the place to add a high level overview of what this method does, and why it exists.